cachi2/core/package_managers/generic.py

```diff
@@ -1,33 +0,0 @@
-from cachi2.core.errors import PackageRejected
-from cachi2.core.models.input import Request
-from cachi2.core.models.output import RequestOutput
-from cachi2.core.models.sbom import Component
-from cachi2.core.rooted_path import RootedPath
-
-DEFAULT_LOCKFILE_NAME = "cachi2_generic.yaml"
-DEFAULT_DEPS_DIR = "deps/generic"
```

Comment:
If you're going to move code, then please do it in a separate commit.

Reply:
Noted, will do so next time that's the case. Sorry about that.